### PR TITLE
Adding state extract for videos

### PR DESCRIPTION
**Pull request by Fausto95**
This PR adds a state-extract function for 
```
state: {
   data: {
     contents: {
        video: {
           entities: {
               id: string
           }
         }
      }
  }
```

### CODE REVIEW

File: packages/@coorpacademy-player-store/src/utils/state-extract.js [hunk 1]
```diff
@@ -493,3 +493,11 @@ export const hasSeenLesson = (state: State, onPreviousSlide: boolean = false): b
...
   );
 };
+
+export const getVideoUrl = (id: string) => (state: State): string =>
+  state && // eslint-disable-line lodash-fp/prefer-get
+  state.data &&
+  state.data.contents &&
+  state.data.contents.video &&
+  state.data.contents.video.entities &&
+  state.data.contents.video.entities[id];
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-player-store/src/utils/state-extract.js]: it seems that it's not in `state.data.contents` but in `state.data` directly:

https://github.com/CoorpAcademy/components/pull/1594/files#diff-df77b9ccae3ea57d6dc59267aae00b5fR48

File: packages/@coorpacademy-player-store/src/utils/test/state-extract.js [hunk 2]
```diff
@@ -866,6 +867,19 @@ test('getNbSlides should return 0 if no contentInfo is found', t => {
   return t.is(getNbSlides(state), 0);
 });
 
+test('getVideoUrl should return the video url', t => {
+  const videoId = 'DE3942dz9ke';
+  const state = set(
+    'data.contents.video.entities',
```
Comment (VincentCATILLON) [inline packages/@coorpacademy-player-store/src/utils/test/state-extract.js]: same than above about the path (`data.contents.video -> `data.video`)

File: packages/@coorpacademy-player-store/src/utils/state-extract.js (hunk 1 above)
Comment (chrisdugne) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: no, it's ok we'll use `fetchContent({type:video})`

Comment (VincentCATILLON) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: yes but the type is incorrect so (see the link above). the video state is not in contents state

Comment (VincentCATILLON) [reply packages/@coorpacademy-player-store/src/utils/state-extract.js]: oh ok it was fixed after my first comment :) my bad

**Pull request merged**